code/game/objects/structures/kitchen_spike.dm

```diff
@@ -9,6 +9,32 @@
 	anchored = FALSE
 	max_integrity = 200
 
+/obj/structure/kitchenspike_frame/Initialize(mapload)
+	. = ..()
+	register_context()
+
+/obj/structure/kitchenspike_frame/examine(mob/user)
+	. = ..()
+	. += "It can be <b>welded</b> apart."
+	. += "You could attach <b>[MEATSPIKE_IRONROD_REQUIREMENT]</b> iron rods to it to create a <b>Meat Spike</b>."
+
+/obj/structure/kitchenspike_frame/add_context(
+	atom/source,
+	list/context,
+	obj/item/held_item,
+	mob/living/user,
+)
+	var/message = null
+	if(held_item.tool_behaviour == TOOL_WELDER)
+		message = "Deconstruct"
+	else if(held_item.tool_behaviour == TOOL_WRENCH)
+		message = "Bolt Down Frame"
+
+	if(isnull(message))
+		return NONE
```

Comment:
```suggestion
	var/message = ""
	if(held_item.tool_behaviour == TOOL_WELDER)
		message = "Deconstruct"
	else if(held_item.tool_behaviour == TOOL_WRENCH)
		message = "Bolt Down Frame"

	if(!message)
		return NONE
```

Declaring the variable as an empty string at the very start acts as a way of "typing" the variable, and will guard against unwanted mutations in case someone in a few years wants to add more context. No need to do null stuff.